test/manifest/capz-template-prow-workload-identity.yaml

```diff
@@ -172,6 +172,7 @@ metadata:
 spec:
   template:
     spec:
+      generation: 2
```

Comment:
I don't think we use this template in test-infra. it should be a change in test-infra repo